.github/workflows/integration_test.yaml

```diff
@@ -0,0 +1,19 @@
+name: Integration tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+
+jobs:
+  integration-tests:
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    secrets: inherit
+    with:
```

Comment:
thought: Can we run these tests with multiple versions of `microk8s` and` `juju` to ensure broader compatibility?

Such a decision probably needs to propagate to other charms we own as well.

Reply:
I think this is good. I also remembered that I had similar discussions with Amber before and we decided that we would leave this type of stuff to the release of a Juju LTS version.

So, if you like it we can keep it and do it over more charms. Otherwise, we can wait until an LTS version and decide on a testing strategy revolving around LTS'.

Reply:
I think it does make more sense to wait on the release of the Juju LTS version.